### DISCUSSION TITLE
Issues with multiple playlists

### DISCUSSION THREAD

**Discussion by Altfel** [AzuraCast/AzuraCast#6546] [General]
Hello!
I have a problem configuring my playlists so I can get the result I want.
The case is I have a main playlist and a secundary one containing a track that I want to be played at the begining of each hour. It's some kind of jingle saying for ex: "Your are listening to ........... It's five o'clock". 
I managed to make it interrupt the main playlist at scheduled time, but the problem is that if main playlist plays any song at that time, after my jingle, the song continues from where it was interrupted. What I really want is to configure it somehow to skip the interrupted song and after my jingle just start whatever is next in the main playlist. 
Could it be possible to configure it like that?
Thank you!

**Comment by Piratenkanon** [comment]
unmark "interrupt the main playlist" and restart AutoDJ service, here works fine and playing after the jingle the next song from the playlist

**Reply by Altfel** [reply]
@Piratenkanon I can't find the option "interrupt the main playlist". In the playlist options - advanced, I have just "Interrupt other songs to play at scheduled time" which is marked (on). I restarted the AutoDJ service, but the behaviour is the same (not skipping current song after the jingle). Can you tell me if there is such option as "interrupt the main playlist" where can I find it? Or if there is another workaround?
My AzuraCast verion is 0.18.5 Stable on Ubuntu 22.04.
Thanks.

**Reply by Piratenkanon** [reply]
Interrupt other songs to play at scheduled time" which is marked (on)  This must be off !!!!!!!   After changing, the AutoDJ must be restarted

**Reply by Altfel** [reply]
I set off that option and restarted AutoDJ, I used as Playlist Type "One per hour" at any given minute.
In this case, the scheduled minute is relative, as it depends on the lenght of the song in the main playlist, which doesn't help me, as I want it to be played at sharp hours, not 1-3 minutes after.
As I had more time, I found another topic posted 2 years ago witch describes exactly my problem, but it seems like it's not such a big issue. The link is: https://github.com/AzuraCast/AzuraCast/issues/2713
